packages/react-reconciler/src/__tests__/ReactSuspenseFuzz-test.internal.js

```diff
@@ -166,17 +166,15 @@ describe('ReactSuspenseFuzz', () => {
           {children}
         </ShouldSuspendContext.Provider>,
       );
-      resolveAllTasks();
+      await resolveAllTasks();
       const expectedOutput = expectedRoot.getChildrenAsJSX();
 
-      gate(flags => {
```

Comment:
👀

Reply:
flags was never used, so this was useless.